atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/LightweightOppTokenTest.java

```diff
@@ -20,41 +20,34 @@
 
 import com.palantir.conjure.java.lib.Bytes;
 import java.nio.ByteBuffer;
-import java.util.Collection;
+import java.util.List;
 import java.util.Locale;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import org.junit.runners.Parameterized.Parameter;
-import org.junit.runners.Parameterized.Parameters;
-
-/* TODO(boyoruk): Migrate to JUnit5 */
-@RunWith(Parameterized.class)
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
 public class LightweightOppTokenTest {
-    @Parameter()
-    public String input;
 
-    @Parameters(name = "{index}: input=\"{0}\"")
-    public static Collection<Object[]> data() {
+    public static List<String> getParameters() {
         return TokensTest.generateTokens()
                 .flatMap(input -> Stream.of(input, input.toLowerCase(Locale.ROOT), input.toUpperCase(Locale.ROOT)))
-                .map(input -> new Object[] {input})
                 .collect(Collectors.toList());
     }
 
-    @Test
-    public void inputsAreValidBase16Strings() {
+    @ParameterizedTest(name = "{index}: input=\"{0}\"")
```

Comment:
Instead of repeating this string all the time, can't we just declare it as static final and reference it everywhere we want to use? Do this for all classes where we're repeating the parameterized test name.

Something like:

```
    private static final String PARAMETERIZED_TEST_NAME = "{index}: input=\"{0}\"";
...

    @ParameterizedTest(name = PARAMETERIZED_TEST_NAME)
```